src/Nancy.Swagger.Annotations/SwaggerObjects/AnnotatedOperation.cs

```diff
@@ -44,33 +44,72 @@ public AnnotatedOperation(string name, MethodInfo handler, ISwaggerModelCatalog
                 .Select(CreateSwaggerResponseObject)
                 .ToDictionary(x => x.GetStatusCode().ToString(), y => (global::Swagger.ObjectModel.Response) y);
 
-            var infos = handler.GetParameters().Where(x => x.GetCustomAttributes<RouteParamAttribute>().Any()).ToList();
+            var paramsList = new List<Parameter>();
+            //first check if any RouteParamAttributes are on the Method.
+            CreateSwaggerParametersFromMethodAttributes(handler, paramsList);
+            //then check for RouteParamAttributes on themethod parameters
```

Comment:
Typo: `the method`

Reply:
I removed all the comments, they were really for my own benefit while working on the change.